packages/itmat-ui-react/src/components/users/userDetails.tsx

```diff
@@ -76,6 +76,7 @@ export const EditUserForm: React.FunctionComponent<{ user: (IUserWithoutToken &
                         </select></label><br /><br />
                     <label>Real name: <input type='text' value={inputs.realName} onChange={e => { setInputs({ ...inputs, realName: e.target.value }) }} /> </label><br /><br />
                     <label>Password:  <input type='password' value={inputs.password} onChange={e => { setInputs({ ...inputs, password: e.target.value }) }} /></label> <br /><br />
+                    <label>OTP Secret (readonly): <input type='text' readOnly value={inputs.otpSecret} /> </label><br /><br />
```

Comment:
I don't think we should bring the OTP secret back to the UI anytime after generation. There is no immediate use for it I shouldn't think. What the intended purpose here ?

Reply:
at the moment I just wanna show it for putting into Google Authenticator app :D. Yeah, for later we can discuss how to send this secret to users :)